### ISSUE TITLE
Switching to dummy 'stops' the game

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#357]
When you switch to a player who is just a dummy, the game just seems to kinda pause. The snail is displayed for the dummy I tried to switch to.
Hitting pause two times doesn't work as well (as in the bug with "fake" pauses)

Version: F8F0B89
OS: Win 7 x64

**Comment by Flamefire** [comment]
Thanks. By dummy you mean the AI that is doing nothing? How can you select that?

**Comment by Spikeone** [comment]
![dummyplayer](https://cloud.githubusercontent.com/assets/1276342/12223303/6ef689d0-b7d3-11e5-9ec7-a0fe7fc00b34.png)

Singleplayer => Unlimited Play => Then switch until you get dummy (just before closed)

**Comment by Flamefire** [comment]
Pretty hard to fix that for good. Why and how is this feature useful?

**Comment by Spikeone** [comment]
If you want to test something and need an enemy who builds something but is then idle.
Instead of using 2 or up to 8 clients you could use dummy players. As far as I remember they were working in 0.7.

**Comment by Flamefire** [comment]
Ah ok. you are right. I used 2 instances for that ;)
The problem is the synchronisation. It is pretty hard to have switch players and keep the game command messages correctly sorted to the correct player. E.g.: Player switch. One message from 1 player was received, but another one is still "on the wire", you switch the 1st message too, but the other one will be received after the switch and hence for the wrong player. And there are a couple of those situations. I eliminated most of them, but there is still a chance that it goes wrong. I think I need a new message for that and rely on TCP to sort things reliably...
